content/attacks/posts/2023-06-06-Harmony’s Horizon.md

```diff
@@ -0,0 +1,33 @@
+﻿---
+date: 2022-06-23
+custodians: Harmony’s Horizon
+categories: Bridge Hack
+title: "Harmony's Horizon Bridge was the victim of a massive cyberattack from North Korea"
+---
+
+## Summary
```

Comment:
Your summary reads more like a timeline. Could you make a one paragraph that summarizes all sections and avoids general definitions, which are more suited for the [docs section](https://github.com/1712n/dn-institute/tree/main/content/attacks/docs) that talks about types of the attacks?